modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -32,22 +32,24 @@ def create
 
       private
 
-      def validated_index_params
```

Comment:
I decided to make this a memoized method because it cleans up the code a little IMO.